repo/locking_storage.go

```diff
@@ -16,7 +17,7 @@ func GetLockingStoragePrefixes() []string {
 	}
 
 	prefixes = append(prefixes, indexblob.V0IndexBlobPrefix, epoch.EpochManagerIndexUberPrefix, format.KopiaRepositoryBlobID,
-		format.KopiaBlobCfgBlobID)
+		format.KopiaBlobCfgBlobID, string(blob.BlobIDPrefixDeleteMarker))
```

Comment:
I added the DM files to be extended but I guess it's not really needed. Doesn't do much harm though. It was for random scenarios like user setting RetentionPeriod larger than the truth and making it possible to delete the DM before the original file is deleted...but as far as kopia is concerned that file would then be orphaned content or a long running session and it'd just attempt to delete again